jekyll/_cci2/server/v4.1/air-gapped-installation/phase-2-configure-object-storage.adoc

```diff
@@ -0,0 +1,126 @@
+---
+version:
+- Server v4.x
+- Server Admin
+---
+= Phase 2 - Configure object storage
+:page-layout: classic-docs
+:page-liquid:
+:page-description: How to configure object storage through MinIO to run CircleCI Server in an air-gapped environment.
+:icons: font
+:toc: macro
+:toc-title:
+
```

Comment:
@circlery @lloydChris I think we could do with a bit of context for this page - just an intro about what the object storage is used for, and why we use minIO. If you could give me some brief notes I can add something.

Reply:
added